libs/blocks/src/lib/platform/card/index.tsx

```diff
@@ -38,7 +38,11 @@ const PlatformBlockCard = ({
       )}
     >
       <FluidContainer className="flex flex-col gap-gap-3xl">
-        {header && <Heading.H2 className="text-center">{header}</Heading.H2>}
+        {header && (
+          <Heading.H2 className="mx-auto max-w-[816px] text-center">
```

Comment:
Hi @jia-deriv just asking is that fine to add hardcoded value?

Reply:
Designer fixed the width in figma to 816px for

Reply:
It's from designers. Since we are using max width so it's won't be a problem..

Reply:
Thank you so much for the update